index.html

```diff
@@ -9,103 +9,161 @@
+                    placeholder="Ex: Jon Craton, Web Apps"
+                    aria-label="Sizing example input"
+                    aria-describedby="inputGroup-sizing-sm"
+                  />
+                </div>
+              </div>
 
-          <div class="col">
-            <select
-              class="form-select form-select-sm"
-              aria-label=".form-select-sm example"
-            >
-              <option selected>Choose a liberal art requirement</option>
-              <option value="1">One</option>
-              <option value="2">Two</option>
-              <option value="3">Three</option>
-            </select>
-          </div>
-          <div class="row">
-            <div class="col">
-              <label for="appt">Select a time:</label>
-              <input type="time" id="appt" name="appt" />
-              <input type="submit" />
-            </div>
-            <div class="col">
-              <div class="form-check form-switch">
-                <input
-                  class="form-check-input"
-                  type="checkbox"
-                  id="flexSwitchCheckDefault"
-                />
-                <label class="form-check-label" for="flexSwitchCheckDefault"
-                  >Show online courses only</label
+              <div class="col-md-auto">
+                <select
+                  class="form-select form-select-sm"
+                  id="select-AU"
+                  aria-label=".form-select-sm example"
                 >
+                  <option selected>Choose a department</option>
+                  <option value="1">One</option>
+                  <option value="2">Two</option>
+                  <option value="3">Three</option>
+                </select>
               </div>
-            </div>
-            <div class="col">
-              <div class="form-check form-switch">
-                <input
-                  class="form-check-input"
-                  type="checkbox"
-                  id="flexSwitchCheckDefault"
-                />
-                <label class="form-check-label" for="flexSwitchCheckDefault"
-                  >Show open courses only</label
+
+              <div class="col-md-auto">
+                <select
+                  class="form-select form-select-sm"
+                  id="select-AU"
+                  aria-label=".form-select-sm example"
                 >
+                  <option selected>Choose a liberal art requirement</option>
+                  <option value="1">One</option>
+                  <option value="2">Two</option>
+                  <option value="3">Three</option>
+                </select>
+              </div>
+              <div class="row">
+                <div class="col-md-auto">
+                  <div class="input-group input-group-sm mb-3">
+                    <span
+                      class="input-group-text classsearch-AU"
+                      id="inputGroup-sizing-sm"
```

Comment:
The id inputGroup-sizing-sm is still reused for multiple elements. Other duplicate ids can be seen here:

https://validator.w3.org/nu/?doc=https%3A%2F%2Fdeploy-preview-61--cpsc4430-course-search.netlify.app%2F

Reply:
> I think the search button is gone?

I also think that the liberal arts drop-down options have changed and the department names are no longer there